src/app_service/service/settings/dto/settings.nim

```diff
@@ -170,6 +175,17 @@ proc toCurrentUserStatus*(jsonObj: JsonNode): CurrentUserStatus =
   discard jsonObj.getProp("clock", result.clock)
   discard jsonObj.getProp("text", result.text)
 
+proc toSettingsFieldDto*(jsonObj: JsonNode): SettingsFieldDto =
+  var field = SettingsFieldDto()
+  field.name = jsonObj["name"].getStr()
+
+  case field.name:
+    of KEY_PROFILE_PICTURES_SHOW_TO:
+      field.value = jsonObj["value"].getInt().intToStr
```

Comment:
This will be extended with more cases as we add handling for other setting fields